### PR TITLE
Issue 245: Rename `window_upr` to `window`

### LINKED ISSUE #245: Rename `swindow_upr` and `pwindow_upr`
I found these names confusing and thought they corresponded to upper bounds i.e. $S_U$ and $P_U$. They are upper bounds, but when you've fixed the $S_L = 0$ and $P_U = 0$.

I think it is more intuitive to think of these as the delay window sizes. As such I'd suggest names like (taking `s` for the time being, with `p` just the same):

1. `swindow`
2. `swindow_width`
3. `swindow_size`

More broadly I think I have been confused by lots of the variable names created in the data. I'll paste them into here and we should decide on names that will be intuitive to users (as if you do e.g. `as_latent_individual` then you'll get all these names).

### PR DESCRIPTION
**Pull request by athowes**
## Description

This PR closes #245. This change is inline with `primarycensordist`.

## Checklist

- [ ] My PR is based on a package issue and I have explicitly linked it.
- [ ] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [ ] I have read the [contribution guidelines](https://github.com/epinowcast/.github/blob/main/CONTRIBUTING.md).
- [ ] I have tested my changes locally.
- [ ] I have added or updated unit tests where necessary.
- [ ] I have updated the documentation if required.
- [ ] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [ ] I have reviewed CI checks for this PR and addressed them as far as I am able.

### CODE REVIEW

File: R/latent_individual.R [hunk 1]
```diff
@@ -85,16 +85,16 @@ epidist_validate.epidist_latent_individual <- function(data) {
     names(data),
     must.include = c("case", "ptime_lwr", "ptime_upr",
                      "stime_lwr", "stime_upr", "obs_at",
-                     "id", "obs_t", "pwindow_upr", "woverlap",
-                     "swindow_upr", "delay_central", "row_id")
+                     "id", "obs_t", "pwindow", "woverlap",
+                     "swindow", "delay_central", "row_id")
```
Comment (athowes) [inline R/latent_individual.R:89]: Is `delay_central` the right name?

File: R/latent_individual.R [hunk 2]
```diff
@@ -85,16 +85,16 @@ epidist_validate.epidist_latent_individual <- function(data) {
     names(data),
     must.include = c("case", "ptime_lwr", "ptime_upr",
                      "stime_lwr", "stime_upr", "obs_at",
-                     "id", "obs_t", "pwindow_upr", "woverlap",
-                     "swindow_upr", "delay_central", "row_id")
+                     "id", "obs_t", "pwindow", "woverlap",
```
Comment (athowes) [inline R/latent_individual.R:88]: Is `obs_t` the right name?

Comment (athowes) [reply R/latent_individual.R:88]: `obs_t` relative versus absolute?

File: R/latent_individual.R (hunk 1 above)
Comment (athowes) [reply R/latent_individual.R:89]: https://github.com/epinowcast/epidist/issues/320

**Review by seabbs — APPROVED** [review]
LGTM

**Pull request merged**